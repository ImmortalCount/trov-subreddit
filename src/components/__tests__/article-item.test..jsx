import React from 'react'
import renderer from 'react-test-renderer'
import ArticleItem from '../article-item'

describe('ArticleItem Component', () => {
    const defaultProps = {
        data: {
            name: 'test name',
            title: 'test title',
            author: 'Emilio Estevez',
            thumbnail: 'self',
            permalink: 'https://example.com',
            created: 154888188
        }
    }

    it('ArticleItem matches snapshot', () => {
        const modalRender = renderer.create(
            <ArticleItem {...defaultProps} />
        ).toJSON()
        expect(modalRender).toMatchSnapshot()
    })
})
