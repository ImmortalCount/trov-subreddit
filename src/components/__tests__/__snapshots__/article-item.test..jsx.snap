// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleItem Component ArticleItem matches snapshot 1`] = `
<li
  className="sc-bwzfXH jzDYmE"
>
  <span
    className="sc-bdVaJa jzEyq"
  >
    no image
  </span>
  <div
    className="sc-bxivhb bUZGRr"
  >
    <h2>
      <a
        href="https://www.reddit.com/https://example.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        test title
      </a>
    </h2>
    <p>
      Posted by 
       
      <a
        href="https://www.reddit.com/user/Emilio Estevez"
        rel="noopener noreferrer"
        target="_blank"
      >
        Emilio Estevez
      </a>
       
      about 49033 years
       
       ago
    </p>
    <p>
      <a
        href="https://www.reddit.com/https://example.com#siteTable"
        rel="noopener noreferrer"
        target="_blank"
      >
        comments
      </a>
    </p>
  </div>
</li>
`;
